test/test_fourier.jl

```diff
@@ -26,9 +26,12 @@ function compare_tables(table1, table2; rtol=0.001, discard = [])
         end
     end
 
-    table1 = Metadata.drop_metadata(table1)
-    table2 = Metadata.drop_metadata(table2)
 
+    data_fields_table1 = filter(field -> !(typeof(field) == Symbol && startswith(string(field), "meta")), names(table1))
+    data_fields_table2 = filter(field -> !(typeof(field) == Symbol && startswith(string(field), "meta")), names(table2))
```

Comment:
This doesn't actually appear to do anything in the test. But it's fine to leave this in. I also don't think `table1` or `table2` seem to have any metadata attached at this point? Either way, with the named tuple approach, you can drop the metadata by just selecting the field of interest. I.e., if you had `table1 = (; results, metadata)` you can just do `table1.results`.